AntistasiOfficial.Altis/Save/fn_loadGame.sqf

```diff
@@ -10,14 +10,6 @@ petros allowdamage false;
 flag_playerList = true;
 publicVariable "flag_playerList";
 
-
-//player
-{
-	[_x] call AS_fnc_loadPlayer;
-} forEach (allPlayers - entities "HeadlessClient_F");
-
-
-
```

Comment:
what was this?

Reply:
So that the HC's wont get tp'ed to a spot near the base on load, you can actually bump into the HC's as they are entities.

Reply:
@StefArma 
This part of code was conflicting with the code from initPlayerLocal.sqf especially when players are not JIP but were in lobby when mission starts (like when to restart server with people active)
Both calls (for the same player) were running in scheduled env, and i assume some race conditions happened. The only place to load player load-out should be `initPlayerLocal.sqf` and executed unscheduled `[player] remoteExecCall ["AS_fnc_loadPlayer",2];` (from start to end in single simulation frame)